src/timelines/preamble.js

```diff
@@ -1,17 +1,56 @@
-import experimentStart from '../trials/experimentStart'
-import startCode from '../trials/startCode'
-import userId from '../trials/userId'
-import holdUpMarker from '../trials/holdUpMarker'
-import { AT_HOME, IS_ELECTRON } from '../config/main'
-
-console.log('at_home', AT_HOME)
-console.log('env at home', process.env.REACT_APP_AT_HOME)
+import { jsPsych } from "jspsych-react";
+import { eventCodes, lang } from "../config/main";
+import holdUpMarker from "../trials/holdUpMarker";
+import { showMessage, userId } from "@brown-ccv/behavioral-task-trials";
+import { config, AT_HOME } from "../config/main";
+console.log("at_home", AT_HOME);
+console.log("env at home", process.env.REACT_APP_AT_HOME);
+console.log(
+  showMessage(config, {
+    responseType: "html_button_response",
+    message: `<h1>${lang.task.name}</h1>`,
```

Comment:
this message is a full stimulus, but the below messages are just text, should this just be `lang.task.name`?

Reply:
Oh missed these! I can remove that console log